x-pack/plugins/aiops/public/components/change_point_detection/chapge_point_detection_page.tsx

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { FC, useCallback } from 'react';
+import {
+  EuiBadge,
+  EuiCallOut,
+  EuiDescriptionList,
+  EuiFlexGrid,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiIcon,
+  EuiPanel,
+  EuiProgress,
+  EuiSpacer,
+  EuiTitle,
+  EuiToolTip,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import type { Query } from '@kbn/es-query';
+import { SearchBarWrapper } from './search_bar';
+import { useChangePontDetectionContext } from './change_point_detection_context';
+import { MetricFieldSelector } from './metric_field_selector';
+import { SplitFieldSelector } from './split_field_selector';
+import { FunctionPicker } from './function_picker';
+import { ChartComponent } from './chart_component';
+
+export const ChangePointDetectionPage: FC = () => {
+  const {
+    isLoading,
+    requestParams,
+    updateRequestParams,
+    annotations,
+    resultFilters,
+    updateFilters,
+    resultQuery,
+  } = useChangePontDetectionContext();
+
+  const setFn = useCallback(
+    (fn: string) => {
+      updateRequestParams({ fn });
+    },
+    [updateRequestParams]
+  );
+
+  const setSplitField = useCallback(
+    (splitField: string) => {
+      updateRequestParams({ splitField });
+    },
+    [updateRequestParams]
+  );
+
+  const setMetricField = useCallback(
+    (metricField: string) => {
+      updateRequestParams({ metricField });
+    },
+    [updateRequestParams]
+  );
+
+  const setQuery = useCallback(
+    (query: Query) => {
+      updateRequestParams({ query });
+    },
+    [updateRequestParams]
+  );
+
+  return (
+    <div data-test-subj="aiopsChanePointDetectionPage">
```

Comment:
Nit: Missing `g` in `aiopsChangePointDetectionPage`,

Reply:
Fixed in https://github.com/elastic/kibana/pull/144093/commits/998cdd4777d337c22db72a311deacb070af587f8